src/polytope_blmos.jl

```diff
@@ -208,7 +256,7 @@ function bounded_compute_inface_extreme_point(
         end
     end
 
-    if sum(a) == sblmo.N
+    if isapprox(sum(a), sblmo.N; atol=atol, rtol=rtol)
```

Comment:
You might have to adapt this function as well. Test some example using `DecompositionInvariantConditionalGradient` as the Frank-Wolfe variant in Boscia.